README.org

```diff
@@ -0,0 +1,82 @@
+#+title: Soffit Authentication Library
+
+* About:
+  uPortal soffits have RESTful backend APIs. This means that when a soffit is running, the backend
+  doesn't have any concept of state - like what user it's talking to. We need to specify in each 
+  request which user is requesting what information. Then it's up to the soffit to verify that the
+  user is who they say they are (/Authentication/), and that they have access to the resources 
+  they're requesting (/Authorization/). 
+  
+  This library was created to externalize the /Authentication/ process. This way, every time we make
+  a soffit we don't have to reinvent the authentication wheel. Also, as added benefits, having
+  this libraries ensures a single, central place to handle changes in the future, as well as ensuring
+  consistent error reporting for every soffit's authentication step.
+
+* Including in a Project:
+  This library was made with convenience in mind. In most cases, only a single method needs to be 
+  called to verify a token, /and/ retrieve the user's personal identifier.
+  
+** Project Setup
+   After following the instructions [[https://code.oakland.edu/ea-developers/training/blob/master/module06/soffits/building-soffits.md][here]] add the following snippets to your =build.gradle=:
```

Comment:
We can probably remove the links to our Gitlab

Reply:
Nice catch